### ISSUE TITLE
Web UI 

### ISSUE THREAD

**Issue by usaaama2010** [YashchenkoN/social-network#8]
Do you have any future plans to build Web UI of this awesome application?

**Comment by YashchenkoN** [comment]
@dev-one yes, I want to build the Web UI of this app, but haven't started yet.
I think it would be great to have Web UI built on top of AngularJS/ReactJS. Unfortunately, I have lack of experience in UI design / bootstrapping, that's why I primarily focus on the backend.

If you want to contribute in this area, you're welcome :) We can create a separate project for WebUI here in Github and create appropriate tickets with descriptions

**Comment by dev-one** [comment]
@YashchenkoN I would love to contribute in this area. I would say building UI with ReactJS would be more good.
